handlers/Home/components/CaseStudy/styles.css

```diff
@@ -2,14 +2,29 @@
 
 .HomeCaseStudy {
   color: #fff;
+  /*
   padding: 5.625em 1.5em;
+  */
   position: relative;
   overflow: hidden;
+  background-image:url('/public/images/cokebg.png');
```

Comment:
this can be removed because it will all come from contentful once the data is all put it.